cli/src/main/package/setup

```diff
@@ -1,7 +1,22 @@
 #!/bin/bash
 
+function createFile() {
+  local fileName=$1
+  echo "File $fileName not found."
```

Comment:
BTW: If we already print out that we created the file in line 7, then this message might only cause additional confusion. WDYT?